test.ipynb

```diff
@@ -232,7 +232,7 @@
   },
```

Comment:
_Commented on notebook `test.ipynb` Cell 4 Line 1_
```python
# add percentage
```
It's not a percentage anymore .....

_Respond and view the context [here](https://gitnotebooks.com/Sara-mibo/Test-review-notebooks/pull/1)._